odk1-src/briefcase-using.rst

```diff
@@ -60,6 +60,19 @@ To download blank forms and completed form instances from an :doc:`Aggregate <ag
 
     However, if your Aggregate server is :doc:`installed on Google App Engine <aggregate-app-engine>`, this setting may cause problems with large attachments. If your form has submission attachments (file uploads, videos, images, sound recordings) you should experiment with this setting and see if it improves or worsens performance.
 
+.. note::
+  :name: briefcase-start-pull-from-last-submission-pulled
+
+  .. rubric:: Start pull from last submission pulled
+
+  .. container:: details
+
+    If your forms have more than 100 submissions, you can benefit from enabling the configuration parameter :guilabel:`Start pull from last submission pulled` in the :guilabel:`Settings` tab to save some requests to your Aggregate server.
```

Comment:
`save some requests to your Aggregate server` is precise, but not user-friendly. What about `speed up pulls` or `dramatically speed up pulls` (if true).

Reply:
Maybe we can steer the phrase toward scenarios with bad network connectivity or server-side costs management. My guess is that the speedup will only be noticeable on huge forms.

Reply:
What about "to improve pull performance on slow connections"?

Reply:
awesome :)